test/integration/elasticquota_controller_test.go

```diff
@@ -55,13 +54,26 @@ func TestElasticController(t *testing.T) {
 	testCtx.ClientSet = cs
 	testCtx.KubeConfig = globalKubeConfig
 
-	schedInformerFactory := schedformers.NewSharedInformerFactory(extClient, 0)
-	eqInformer := schedInformerFactory.Scheduling().V1alpha1().ElasticQuotas()
+	s := scheme.Scheme
+	runtime.Must(schedv1alpha1.AddToScheme(s))
 
-	coreInformerFactory := informers.NewSharedInformerFactory(cs, 0)
-	podInformer := coreInformerFactory.Core().V1().Pods()
-	eqCtrl := controller.NewElasticQuotaController(cs, eqInformer, podInformer, extClient)
-	runtime.Must(schedv1alpha1.AddToScheme(scheme.Scheme))
+	mgrOpts := manager.Options{
+		Scheme:             s,
+		MetricsBindAddress: "0", // disable metrics to avoid conflicts between packages.
+	}
+	mgr, err := ctrl.NewManager(globalKubeConfig, mgrOpts)
+	if err = (&controllers.ElasticQuotaReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+	}).SetupWithManager(mgr); err != nil {
+		t.Fatal("unable to create controller", "controller", "PodGroup", err)
```

Comment:
minor comment: Is it ElasticQuota, not PodGroup?

Reply:
good catch, @czybjtu